src/TodoInfo.tsx

```diff
@@ -0,0 +1,119 @@
+import classnames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+
+type Props = {
+  todo: Todo,
+  onDelete: (id: number) => void,
+  onStatusChange: (todo: Todo) => void,
+  onTitleChange: (todo: Todo, todoTitle: string) => void,
+  isLoading: boolean,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  onDelete,
+  onStatusChange,
+  onTitleChange,
+  isLoading,
+}) => {
+  const { id, title, completed } = todo;
+
+  // const [isBeingDeleted, setIsBeingDeleted] = useState(false);
+  const [titleChanger, setTitleChanger] = useState(false);
+  const [newTitle, setNewTitle] = useState(title);
+  const [focusDisable, setFocusDisable] = useState(false);
```

Comment:
Get used to name your boolean state according to these rules
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig